configs/cityscapes/README.md

```diff
@@ -0,0 +1,33 @@
+# Benchmark and Model Zoo
+
+## Environment
+
+### Hardware
+
+- 1 NVIDIA GeForce GTX 1080 Ti GPU
+
+## Common settings
+
+- All baselines were trained using 1 GPU with a batch size of 2 (2 images per GPU) using the [linear scaling rule](https://arxiv.org/abs/1706.02677) to scale the learning rate. The learning rate in the configs is set for a batch size of 16 to match the default of the coco models.
```

Comment:
8 GPUs now.

Reply:
Can you just add in your setup descriptions?